inc/toolbox.class.php

```diff
@@ -1741,6 +1741,19 @@ static function return_bytes_from_ini_vars($val) {
       return $val;
    }
 
+
+   /**
+    * Get max upload size from php config.
+    *
+    * @return int
+    */
+   static function getPhpUploadSizeLimit(): int {
```

Comment:
maybe add a unit test on that one too

Reply:
I did not find a solution for this.

I have to mock the `ini_get` function, but I have this error `Error: The function you are trying to mock already exists: 'ini_get'. This may be because you are trying to mock a function from a class in the root namespace.`.

If I try to change the put my test in a sub namespace, I have this error `Tested class 'Glpi\Toolbox' does not exist for test class 'tests\units\Glpi\Toolbox'`.